### ISSUE TITLE
Many of the variable names are incomprehensible and confusing

### ISSUE THREAD

**Issue by veryeli** [Khan/guacamole#3]

**Comment by shobhitverma** [comment]
bump

**Comment by pablocelayes** [comment]
I agree. In particular in https://github.com/Khan/guacamole/blob/master/mirt/mirt_util.py line 185, I think it should be `discriminations` instead of `difficulties`, right?

Also the arguments in the docstring for method `conditional_probability_correct` don't match the actual arguments.
